src/s1reader/s1_burst_slc.py

```diff
@@ -173,6 +175,10 @@ class Sentinel1BurstSlc:
     burst_noise: s1_annotation.BurstNoise  # Thermal noise correction
     burst_eap: s1_annotation.BurstEAP  # EAP correction
 
+    # Time series of FM rate / Doppler centroid polynomial coefficients
+    # for azimith FM rate mismatch mitigation
+    extended_coeffs: s1_annotation.BurstExtendedCoeffs
```

Comment:
We do extrapolate the coefficients, but I think we just keep one. Do you need to extract all of them?

Reply:
@vbrancat Can you show me the link to the examples where we do the extrapolation? That would be very helpful for me to  understand if that can be usable for the FM rate mismatch.

We need a sequence of the polynomials that is enough to cover the azimuth time of a burst. That is because we interpolate the polynomials along the azimuth time (of a burst), so usually 2~3 sets of polynomials are needed.

Reply:
Thanks @vbrancat for the snippets. I've revised the code to re-use the parsed coefficients (i.e. `az_fm_rate_list` and `doppler_list`) and extract the sequence that covers the burst's azimuth time.